libs/auth-api-lib/src/lib/resources/admin/admin-scope.service.ts

```diff
@@ -459,17 +459,33 @@ export class AdminScopeService {
     )
 
     // create delegation type rows
-    await Promise.all(
-      delegationTypes.map((delegationType) =>
-        this.apiScopeDelegationType.upsert(
-          {
-            apiScopeName,
-            delegationType,
-          },
-          { transaction },
-        ),
-      ),
-    )
+    delegationTypes.map((delegationType) => {
```

Comment:
You should do the same here just check if the array includes custom and construct the new shallow copied array with GeneralMandate added.